salt/addons/dex/legacy.sls

```diff
@@ -0,0 +1,24 @@
+include:
+  - .main
```

Comment:
Is this `legacy.sls` invoked from anywhere? I cannot find any reference...

Reply:
Ah - I forgot to add it back into `init.sls`.

I am still hoping to find a good way to not having to show the dependencies in the state files *and* the orchestration (`orch.kubernetes`) as well.